README.md

```diff
@@ -1799,13 +1786,53 @@ HTTP-response status code:
 
 201 created (empty response body)
 
+### 4.5.7 GET bitmap of a Viewpoint Service
+
+**Resource URL**
+
+	GET /bcf/{version}/projects/{guid}/topics/{guid}/viewpoints/{guid}/bitmaps/{guid}
+
+Retrieve a specific viewpoint bitmaps image file (png, jpg or bmp).
+
+**Example Request**
+    https://example.com/bcf/1.0/projects/F445F4F2-4D02-4B2A-B612-5E456BEF9137/topics/B345F4F2-3A04-B43B-A713-5E456BEF8228/viewpoints/a11a82e7-e66c-34b4-ada1-5846abf39133/bitmaps/760bc4ca-fb9c-467f-884f-5ecffeca8cae
+
+**Example Response**
+
+HTTP-response header:
+	
+	Content-Type: image/png
+
+### 4.5.8 PUT bitmap of a Viewpoint Service
+
+**Resource URL**
+
+	PUT /bcf/{version}/projects/{guid}/topics/{guid}/viewpoints/{guid}/bitmaps/{guid}
+
+Add or update a specific bitmap in a viewpoint (png, jpg or bmp).
```

Comment:
I suggest removing the add functionality in the PUT method, and creating a new resource that creates a bitmap in the given viewpoint: POST /bcf/{version}/projects/{guid}/topics/{guid}/viewpoints/{guid}/bitmaps

Also I don't see any way to specify the bitmap properties such as location, normal and up. Could those properties be appended to the PUT/POST resources? Will it work to upload images together with other body properties?